lib/bundler.js

```diff
@@ -96,7 +96,7 @@ function bundle(originPointer, originSchema, externalSchemaLoader) {
         else {
             for (var property in schemaPart) {
                 if (schemaPart.hasOwnProperty(property)) {
-                    if (typeof schemaPart[property] === 'object') {
+                    if (typeof schemaPart[property] === 'object' && schemaPart[property] !== null) {
```

Comment:
nit: all conditions can be combined into one `if`?

Reply:
I second that